src/main/java/com/codeborne/selenide/SelenideTargetLocator.java

```diff
@@ -62,7 +73,11 @@ public WebElement activeElement() {
 
   @Override
   public Alert alert() {
-    return Wait().until(alertIsPresent());
+    try {
+      return Wait().until(alertIsPresent());
+    } catch (TimeoutException e) {
+      throw new NoAlertPresentException();
```

Comment:
Don't forget to wrap the original exception: `throw new NoAlertPresentException(e);`